examples/nas/gaea_pytorch/eval/distributed_no_gcp.yaml

```diff
@@ -0,0 +1,85 @@
+# NOTE: this config uses randomly generated data on single GPU.
```

Comment:
non-blocking: maybe rename this file to `distributed_no_data_downaload.yaml` or something like that.